x-pack/plugins/cloud_security_posture/public/pages/rules/rules_bottom_bar.tsx

```diff
@@ -16,7 +16,15 @@ interface RulesBottomBarProps {
 }
 
 export const RulesBottomBar = ({ onSave, onCancel, isLoading }: RulesBottomBarProps) => (
-  <EuiBottomBar>
+  <EuiBottomBar
+    style={{
+      // same as .euiFlyout
+      zIndex: 1000,
+    }}
+    usePortal
+    position="fixed"
+    affordForDisplacement={false}
```

Comment:
Not sure this is the best behavior, if the table has many rules some might get clipped

<img width="1840" alt="image" src="https://user-images.githubusercontent.com/94899776/159244680-6d9fa583-c3a3-4f5b-b339-256f378efd7b.png">

That being said I can't make it add padding to the body, even when setting this to `true` - worth taking a look at

Reply:
do you mean having the bottom bar not on top of the table at all?

Reply:
Yes to have padding on the body like the [EUI docs](https://elastic.github.io/eui/#/layout/bottom-bar#displacement) specify

Reply:
that eui guide says there' a prop that does that, not when to use it. 

i intentionaly set it to false, because i found the UX to be annoying: 

https://user-images.githubusercontent.com/20814186/159918634-b9a548f2-688e-498c-a61a-d591cfea326c.mov